src/pages/Admin/AdminEventModal.tsx

```diff
@@ -214,8 +214,8 @@ const AdminEventModal: React.FunctionComponent<PropsType> = ({
                 ref={register}
                 value={roomName}
               />
-              {errors.host && (
-                <span className="input-error">{errors.host.message}</span>
+              {errors.room && (
```

Comment:
Looks like `room` does not have any validation requirements. So we can remove this block `{errors.room && ...}` fully. 
But ideally would be to add a validation rule allowing only existent room names. But probably it's out of scope of this PR.

Reply:
I'll merge this and make a new PR for that